datadog/resource_datadog_synthetics_test_.go

```diff
@@ -819,6 +826,164 @@ func syntheticsTestOptionsList() *schema.Schema {
 	}
 }
 
+func syntheticsMobileTestOptionsList() *schema.Schema {
+	return &schema.Schema{
```

Comment:
Maybe later we could reuse the `syntheticsTestOptionsList`  and add just the fields we need?
```
schema := syntheticsTestOptionsList()
schema.Elem.Schema["allow_application_crash"] = &schema.Schema{
  Type:     schema.TypeBool,
  Optional: true,
}
```

Reply:
Yes that would be nice :D